pipelines/common/src/main/java/com/google/fhir/analytics/DwhFiles.java

```diff
@@ -197,12 +221,10 @@ public Set<String> findNonEmptyFhirResourceTypes() throws IOException {
    * @throws IOException
    */
   public void copyResourcesToDwh(String resourceType, DwhFiles destDwh) throws IOException {
-    ResourceId sourceFiles =
-        FileSystems.matchNewResource(getRoot(), true)
-            .resolve(resourceType, StandardResolveOptions.RESOLVE_DIRECTORY)
-            .resolve("*", StandardResolveOptions.RESOLVE_FILE);
-
-    List<MatchResult> sourceMatchResultList = FileSystems.matchResources(List.of(sourceFiles));
+    String fileSeparator = getFileSeparatorForDwhFiles(getRoot());
+    List<MatchResult> sourceMatchResultList =
```

Comment:
Have you tested your changes on a GCS example? If yes, please consider updating PR description ("E2E test" field) too.

Reply:
Yes, tested for GCS on a Unix system. Updated the description as well.